docs/conf.py

```diff
@@ -25,41 +25,35 @@
 import os
 import sys
 import warnings
+from datetime import datetime, timezone
+
+from sphinx.application import Sphinx
+
+from plasmapy import __version__
 
 # isort: off
 sys.path.insert(0, os.path.abspath(".."))  # noqa: PTH100
 sys.path.insert(0, os.path.abspath("."))  # noqa: PTH100
 # isort: on
 
-from datetime import datetime
-
 import _cff_to_rst
-import pkg_resources  # deprecated; after removal, drop setuptools dependency for docs
 from _global_substitutions import global_substitutions
-from sphinx.application import Sphinx
-
-# Generate author list from CITATION.cff
-
-_cff_to_rst.main()
-
-from plasmapy import __version__ as release
 
 # Project metadata
 
 project = "PlasmaPy"
 author = "PlasmaPy Community"
-copyright = f"2015–{datetime.utcnow().year}, {author}"  # noqa: A001, DTZ003
+copyright = f"2015–{datetime.now(timezone.utc).year}, {author}"  # noqa: A001
 language = "en"
+release = __version__
+version = __version__
 
 if release.startswith("0"):
     warnings.warn(f"Incorrect version in documentation build ({release = })")
 
-release = "" if release == "unknown" else release
-parsed_version = pkg_resources.parse_version(release)  # deprecated
-release = parsed_version.public
-version = ".".join(release.split(".")[:2])  # short X.Y version
-revision = parsed_version.local[1:] if parsed_version.local is not None else ""
```

Comment:
Previously, `release` was supposed to be the full version and `version` was supposed to be the `X.Y` without patch version.  But, there isn't really a need for us to make a distinction between them.